### DISCUSSION TITLE
how to use file check file name command fx("fname") TASMOTA

### DISCUSSION THREAD

**Discussion by royan204** [arendst/Tasmota#20699] [Q&A] [answered]
Hi, I'm currently working on a project using the RC522 RFID sensor. I want to save its UID to a directory containing UID files. Now, I need the command fx("fname") to search for a matching UID from files inside the directory when a UID comes in from RC522, but I don't know how to use the fx("fname") command.

**Comment by royan204** [comment, marked as answer]
nvm i found it myself

```
>D
uid=""

>E
uid=RC522#UID
if uid!="" { 
	if 
		fx(uid)==1

	{
		=>power1 1
		delay(300)
		=>power1 0
	} else {
		=>power1 1
		delay(100)
		=>power1 0
		delay(100)
		=>power1 1
		delay(100)
		=>power1 0
	}
	uid=""
} 

>R
```

**Comment by Noschvie** [comment]
@royan204 have you tried to use Berry for your project?

**Reply by royan204** [reply]
Yes, I've tried using a berry for my project, but in this situation, I want to use ESP8266, which only supports scripting.
